app/assets/javascripts/discourse/app/components/sidebar.hbs

```diff
@@ -2,10 +2,10 @@
   <div class="sidebar-scroll-wrap">
     {{#if this.currentUser}}
       <Sidebar::Sections @collapsableSections={{true}}/>
-      <Sidebar::Footer />
     {{else}}
       <Sidebar::AnonymousSections />
-      <Sidebar::Footer />
     {{/if}}
   </div>
+
+  <Sidebar::Footer />
```

Comment:
I am going to assume that there would be proper guards in place in the footer for absence of currentUser

Reply:
The footer is displayed regardless for both anon and logged in users.